cross_val.py

```diff
@@ -107,66 +105,59 @@ def _train(self, criterion, optimizer, fold_idx):
                 optimizer.step()
 
                 if global_step % self.opt.log_step == 0:
-                    n_correct += (torch.argmax(outputs, -1) == targets).sum().item()
-                    n_total += len(outputs)
-                    train_acc = n_correct / n_total
 
                     # switch model to evaluation mode
                     self.model.eval()
-                    test_acc, f1 = self._evaluate_acc_f1(val_data_loader)
-                    if test_acc > max_test_acc:
-                        max_test_acc = test_acc
-                        if not os.path.exists('state_dict'):
-                            os.mkdir('state_dict')
-                        path = 'state_dict/{0}_{1}_acc{2}'.format(self.opt.model_name, self.opt.dataset, round(test_acc, 4))
-                        torch.save(self.model.state_dict(), path)
-                        print('>> saved: ' + path)
-                    if f1 > max_f1:
-                        max_f1 = f1
+                    val_loss = self._evaluate_loss(criterion, val_data_loader)
+                    val_loss_list.append(val_loss)
```

Comment:
I think we can simply perform validation at the end of each epoch, like [this](https://github.com/howardhsu/BERT-for-RRC-ABSA/blob/4477a904ee1f821d0d3c0515254c859c6854ee3c/src/run_asc.py#L125).